stdlib/SparseArrays/src/linalg.jl

```diff
@@ -1,12 +1,32 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
-import LinearAlgebra: checksquare
+import LinearAlgebra: checksquare, sym_uplo
 using Random: rand!
 
 # In matrix-vector multiplication, the correct orientation of the vector is assumed.
 const StridedOrTriangularMatrix{T} = Union{StridedMatrix{T}, LowerTriangular{T}, UnitLowerTriangular{T}, UpperTriangular{T}, UnitUpperTriangular{T}}
 const AdjOrTransStridedOrTriangularMatrix{T} = Union{StridedOrTriangularMatrix{T},Adjoint{<:Any,<:StridedOrTriangularMatrix{T}},Transpose{<:Any,<:StridedOrTriangularMatrix{T}}}
 
+for (Wrapper, conjugation) ∈ [(:Hermitian, :adjoint), (:Symmetric, :transpose)]
```

Comment:
```suggestion
for Wrapper ∈ (:Hermitian, :Symmetric)
```
Seems like you don't need the `conjugation` here...

Reply:
Yep, that's left over from before, thanks.